ThePensionsRegulator.Frontend/Styles/_tpr-file-upload.scss

```diff
@@ -0,0 +1,36 @@
+﻿.govuk-file-upload {
+    font-size: 1rem;
+    outline: 3px solid transparent;
```

Comment:
Use variables rather than hard-coding. `3px` here should be `$govuk-focus-width`

Reply:
This should be `$govuk-focus-width` because it's here so that it can be made visible on focus.

Reply:
changed as discussed and suggested